enterprise-suite/alertmanager/alertmanager.yml

```diff
@@ -0,0 +1,55 @@
+# Example Lightbend Console Alertmanager config.
+
+global:
+
+# Routing rules.
+route:
+  receiver: slack
+  # Resend unresolved alerts every 4h.
+  repeat_interval: 4h
+  # Wait 30s for any other alerts in the same group.
+  group_wait: 30s
+  # Wait at least 5m before sending a new alert for a group.
+  group_interval: 5m
+  # Group alerts by workload - will only alert once per workload.
+  group_by: [namespace, es_workload]
+  routes:
+  - match:
+      severity: warning
+    receiver: slack
+  - match:
+      severity: critical
+    receiver: slack
+    continue: true
+  - match:
+      severity: critical
+    receiver: pagerduty
+
+# Don't alert on warnings if the same alert is already critical.
+inhibit_rules:
+- source_match:
+    severity: critical
+  target_match:
+    severity: warning
+  equal: [alertname]
+
+templates:
+# Slack notification templates.
+- '/etc/config/slack.tmpl'
+
+# Receivers.
+receivers:
+- name: slack
+  slack_configs:
+  # Use Slack integrations to find the webhook.
+  - api_url: https://slack.localhost/services/somewhere
```

Comment:
Does this need to be templatized or is that an actual value?

Reply:
This file isn't intended to be used as-is - see my docs change https://github.com/lightbend/es-docs/pull/55. It's an example that users should copy and edit, then load into their own configmap.